src/jazkarta/easyformplugin/savecontent/browser.py

```diff
@@ -35,3 +35,9 @@ def updateWidgets(self):
 
 class FormContentFolderListingView(FolderView):
     """Form submissions listing page with a customized template"""
+
+    def getTitle(self):
```

Comment:
Why not just override `Title()` here? That's the standard method, and should avoid the need for the listing customization.

Reply:
@alecpm I renamed the `getTitle` method to `Title` for consistency, but we'll still need the listing customization.

Turns out that we actually need also the `content-core` block to be customized due to [this change](https://github.com/jazkarta/jazkarta.easyformplugin.savecontent/pull/4/files#diff-ebb959d44f169428a1831f43fef00446ce9e546ff285bead956540512980e866R44).